### ISSUE TITLE
LIST() and HASH() don't have the rigth SIZE(/n_dim)

### ISSUE THREAD

**Issue by alaingdl** [gnudatalanguage/gdl#1725]
Thanks to some super simple tests in error in `test_isa.pro`, I got : 

```
GDL> size(hash(),/n_di)
           1
GDL> size(list(),/n_di)
           1

[...]
IDL> size(hash(),/n_di)
           0
IDL> size(list(),/n_di)
           0
```

**Comment by alaingdl** [comment]
This one is probably solved, but the side effect of large blind tests is I found new problems in ISA()

**Comment by alaingdl** [comment]
Should be closed in #1786 
Please open a new issue if related problem emerge ...
